.platform/services.yaml

```diff
@@ -1,5 +1,5 @@
 mysqldb:
-    type: mysql:10.1
+    type: mysql:10.2
```

Comment:
Why updating this? 
Master is still on `10.1`, I would recommend to not change the version
https://github.com/ezsystems/ezplatform/blob/master/.platform/services.yaml#L2

Reply:
This should be added to master repository -it's the solution for keys length for utf8mb4 for platform.sh

Reply:
Since this has been updated on master, I think we are good to go. 
Could you please rebase this PR?